static/css/theme-activestate.css

```diff
@@ -5,20 +5,20 @@
 
 :root{
```

Comment:
It looks like we're specifying font-family changes here AND in the theme above. Is that necessary? Can we just specify the headings and fonts here and not modify the theme above when we don't have to?